### PR TITLE
[network] Removed last references to org.apache...

### PR DESCRIPTION
**Pull request by clinique**
...  and a bit of code cleansing.

Signed-off-by: Gaël L'hopital <gael@lhopital.org>

### CODE REVIEW

File: bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/NetworkUtils.java [hunk 1]
```diff
@@ -173,16 +202,22 @@
...
-            // We cannot estimate the command line for any other operating system and just return false
+        if (os == null) {
             return IpPingMethodEnum.JAVA_PING;
+        } else {
+            os = os.toLowerCase();
+            if (os.indexOf("win") >= 0) {
+                method = IpPingMethodEnum.WINDOWS_PING;
+            } else if (os.indexOf("mac") >= 0) {
+                method = IpPingMethodEnum.MAC_OS_PING;
+            } else if (os.indexOf("nix") >= 0 || os.indexOf("nux") >= 0 || os.indexOf("aix") > 0) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/NetworkUtils.java:215]: Can you confirm that last condition is > 0 and not >= 0 ?

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
@mettke : would you like to review this PR?

**Review by jlaur — CHANGES_REQUESTED** [review]
Please see comment about typo.

**Review by mettke — APPROVED** [review]
LGTM too

**Comment by cweitkamp** [comment]
Is it possible to clean-op the pom right now?

**Comment by clinique** [comment]
> Is it possible to clean-op the pom right now?

Sure, but what does it mean ?

**Comment by lolodomo** [comment]
I believe it means this:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.network/pom.xml#L35

**Comment by clinique** [comment]
> I believe it means that: https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.network/pom.xml#L35

Unfortunately, these are still required for the speedtest part of the binding

**Comment by cweitkamp** [comment]
Yes, that was exactly what I meant. Thanks for looking into it. If we still need it we do not have to clean it.

File: bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java [hunk 2]
```diff
@@ -608,22 +611,18 @@ public void stopAutomaticRefresh() {
...
-            } else {
-                dhcpState = "Running normally";
-            }
+            DHCPPacketListenerServer listener = DHCPListenService.register(destinationAddress.getHostAddress(), this);
+            dhcpState = String.format("Bound to port %d - %s", listener.getCurrentPort(),
+                    (listener.usingPrivilegedPort() ? "Running normally" : "Port forwarding necessary !"));
         } catch (SocketException e) {
-            logger.warn("Cannot use DHCP sniffing.", e);
+            dhcpState = String.format("Cannot use DHCP sniffing: %s", e.getLocalizedMessage());
+            logger.warn("{}", dhcpState);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java:619]: Why getLocalizedMessage ?
You are potentially mixing a text in English and a translated text.
And for logging, the recommendation is to remain in English.
Translation is only for what can be displayed in UI.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java (hunk 2 above)
Comment (clinique) [reply bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java:619]: Sorry for the delay @lolodomo , I was working on the netatmo binding that absorbed my time.

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**